app/models/cart.rb

```diff
@@ -19,6 +19,10 @@ def total # use reduce
     contents.each { |k, v| total += v["beer_price"].to_i }
     total
   end
+  
+  def delete_item(beer_id)
+    contents.delete_if { |k, v| k == beer_id }
```

Comment:
Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.